### ISSUE TITLE
Full-text search results are not fully navigable

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#4839]
### Describe the inaccessible feature
On all themes, when `ENABLE_FULL_TEXT_SEARCH` is enabled in DJango-level settings, a template is used (`journal/full-text-search.html`) that is not accessible.

* No page heading (h1) is provided
* No pagination for results list
* No indicator of how many results were returned

### Which users are impacted?
 2. Readers (i.e. those accessing the published site but not logging in to Janeway)

### Janeway version
1.7.2

### To Reproduce
Steps to reproduce:
1. Turn on ENABLE_FULL_TEXT_SEARCH and generate indexes
2. Run an article search
3. See page

### Screenshots
![Screenshot of inaccessible article search results](https://github.com/user-attachments/assets/7947015a-fb4c-4b85-9f57-41fbbb930794)

## Preferred Solution
Add headings, pagination, and a result count, to mirror the search page you get when ENABLE_FULL_TEXT_SEARCH is turned off

## Additional context
Created as an offshoot from https://github.com/openlibhums/janeway/issues/4242
